### DISCUSSION TITLE
Super Notes: Customizing column sizes in tables to increase readabilty

### DISCUSSION THREAD

**Discussion by user4-33** [standardnotes/forum#3399] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes.

**Is your feature request related to a problem? Please describe.**
When creating a table and inserting good amounts of text in every column, the table will "shrink" itself to fit the screen width, meaning that every column will have a maximum screen size of `1/total amount of columns`. This really harms the readability of the given text. Also when adding new columns the scaling will continue, decreasing the readability even more.

**Describe the solution you'd like**
The ability to set the column size for each column individually, meaning that the column would expand over the given screen width and then can be horizontally scrolled if the table width would resolve bigger than screen width.

**Describe alternatives you've considered**
/

**Additional context**
Screenshot of the problem:
<img width="984" alt="supernotes_columns" src="https://github.com/standardnotes/forum/assets/119926454/8ba4bdb8-4f85-4b8a-a939-b207ff1e921a">

**Comment by kleters** [comment]
Also interested in customizable column sizes.

I am thinking of a table with only two columns, where the first column should be smaller, while the second column needs to be bigger.

**Comment by aldem** [comment]
+1

Also encountered this issue and was unpleasantly surprised that columns could not be resized - this is a basic feature.

**Reply by edmundloo** [reply]
> +1
> 
> Also encountered this issue and was unpleasantly surprised that columns could not be resized - this is a basic feature.

I'm on the same page here. Would love to see this happen.

**Comment by Castijn537** [comment]
+1
